test/ring/logger_test.clj

```diff
@@ -123,6 +123,20 @@
            (->> (map #(nth % 3) @*entries*)
                 (filter #(re-find #"secret" %)))))))
 
+(deftest basic-ok-request-logging-without-query-params
```

Comment:
Thanks for adding this test!

I don't think we are testing the opposite (and default) case where `query-string` is logged anywhere else. It would be great if you could add a test to show that query string is logged out when `log-query-string?` is not set (so it takes the default value). It could have a param from the set of redact-keys, like `password` for example, to show it goes through unredacted

Reply:
I've added new test which covers default settings (`log-query-string?` not being set)